tower/src/make/mod.rs

```diff
@@ -1,7 +1,9 @@
 //! Trait aliases for Services that produce specific types of Responses.
 
 mod make_connection;
-mod make_service;
+
+pub mod make_service;
 
 pub use self::make_connection::MakeConnection;
+#[doc(inline)]
 pub use self::make_service::MakeService;
```

Comment:
Sorry, I misunderstood what you were saying earlier. This is the `tower::make` module, which is public, and contains a private module named `make_service`. I thought you were saying the `make` module itself was not public.

Let's not make `tower::make::make_service` pub. Let's just re-export `AsService` and `IntoService` _in_ the `tower::make` module:
```suggestion
mod make_service;

pub use self::make_connection::MakeConnection;
pub use self::make_service::{MakeService, AsService, IntoService};
```

I think this is fine --- it's clear that these are `MakeService` adapters because they're in the `tower::make` module.

Reply:
Haha all good